src/legacy/core_plugins/kibana/public/dashboard/store/panel_actions_store.ts

```diff
@@ -28,7 +28,9 @@ class PanelActionsStore {
    */
   public initializeFromRegistry(panelActionsRegistry: ContextMenuAction[]) {
     panelActionsRegistry.forEach(panelAction => {
```

Comment:
Some reason panel actions are duped everytime you re-enter a page when client-side routing occurs. This fixes that.